frontend/src/action-creator.async.js

```diff
@@ -455,9 +455,9 @@ export const getWorkspaceDetail = (user, workspaceId) => dispatch => {
   })
 }
 
-export const getWorkspaceMemberList = workspaceId => dispatch => {
+export const getWorkspaceMemberList = (workspaceId, showDisabledUser = false) => dispatch => {
   return fetchWrapper({
-    url: `${FETCH_CONFIG.apiUrl}/workspaces/${workspaceId}/members`,
+    url: `${FETCH_CONFIG.apiUrl}/workspaces/${workspaceId}/members?show_disabled_user=${showDisabledUser ? 1 : 0}`,
```

Comment:
since the endpoint has a default value for show_disabled_user to false, I would prefer not putting any query param instead of setting it to 0